src/data_storage/import/type_transformation_storage.ts

```diff
@@ -0,0 +1,141 @@
+import Result from "../../result"
+import PostgresStorage from "../postgresStorage";
+import {QueryConfig} from "pg";
+import PostgresAdapter from "../adapters/postgres/postgres";
+import {dataSourceT, DataSourceT} from "../../types/import/dataSourceT";
+import {typeTransformationT, TypeTransformationT} from "../../types/import/typeMappingT";
+
+/*
+* ImportAdapterStorage encompasses all logic dealing with the manipulation of the Import Adapter
+* class in a data storage layer.
+*/
+export default class TypeTransformationStorage extends PostgresStorage{
+    public static tableName = "data_type_mapping_transformations";
```

Comment:
Perhaps an overly verbose table name?

Reply:
going to keep the table name as its descriptive and shows the relationship between mappings and transformations. Also helps in the join statements